site/content/docs/5.1/components/navs-tabs.md

```diff
@@ -172,6 +172,62 @@ Nav tabs light only differ visually, with a full width bottom border and a diffe
 {{< /example >}}
 <!-- End mod -->
 
+### Nested tabs
+
+
```

Comment:
Extra line to remove. It is probably due to the extra line also under `### Tabs light` which should be removed as well.

Reply:
Done for both